### PR TITLE
Update German (de) translation

### PR DESCRIPTION
**Pull request by daPhipz**
I added some new strings and reworded existing translations that make more sense this way IMO.

Feel free to get them proofread by some other German contributor, since this is my first contribution to this project :)

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -25,39 +26,50 @@ msgstr "GSConnect"
...
 
 #: data/metainfo/org.gnome.Shell.Extensions.GSConnect.metainfo.xml.in:32
-msgid "GSConnect is a complete implementation of KDE Connect especially for GNOME Shell with Nautilus, Chrome and Firefox integration. The KDE Connect team has applications for Linux, BSD, Android, Sailfish, macOS and Windows."
+msgid ""
+"GSConnect is a complete implementation of KDE Connect especially for GNOME "
+"Shell with Nautilus, Chrome and Firefox integration. The KDE Connect team "
+"has applications for Linux, BSD, Android, Sailfish, macOS and Windows."
 msgstr ""
+"GSConnect ist eine vollständige Implementierung von KDE Connect für GNOME "
+"Shell, insbesondere mit Nautilus-, Chrome- und Firefos-Integration. Das KDE "
```
Comment (slrtbtfs) [inline po/de.po:42]:
```suggestion
"Shell, insbesondere mit Nautilus-, Chrome- und Firefox-Integration. Das KDE "
```

File: po/de.po [hunk 2]
```diff
@@ -239,7 +251,7 @@ msgstr "Dateien empfangen"
 
 #: data/ui/preferences-device-panel.ui:490
 msgid "Save files to"
-msgstr ""
+msgstr "Dateien speichern nach"
```
Comment (slrtbtfs) [inline po/de.po:254]:
```suggestion
msgstr "Dateien speichern unter"
```
AFAIK, this wording is more common in German software translations.

File: po/de.po [hunk 3]
```diff
@@ -1101,13 +1084,13 @@ msgstr "Sie: %s"
...
 msgstr[0] "Und %d anderer Kontakt"
-msgstr[1] "Und %d Andere"
+msgstr[1] "Und %d andere"
 
 #. TRANSLATORS: Displayed when the remote keyboard is not ready to accept input
 #: src/service/ui/mousepad.js:110
 #, javascript-format
 msgid "Remote keyboard on %s is not active"
-msgstr ""
+msgstr "Entfernte Tastatur auf %s ist nicht aktiv"
```
Comment (slrtbtfs) [inline po/de.po:1093]:
```suggestion
msgstr "Tastaturfernsteuerung auf %s ist nicht aktiv"
```

**Review by slrtbtfs — COMMENTED** [review]
I'm not a contributor to this project yet, but I do speak German natively, so I've had a look at the translated strings.

Except for where I've added comments, everything looked good to me.

File: po/de.po (hunk 1 above)
Comment (daPhipz) [reply po/de.po:42]: Oops, yes. What is Firefos? 😂😂

File: po/de.po (hunk 2 above)
Comment (daPhipz) [reply po/de.po:254]: Yep, you are right.

File: po/de.po (hunk 3 above)
Comment (daPhipz) [reply po/de.po:1093]: Yes, this sounds good IMO.

**Comment by daPhipz** [comment]
Thanks for your input!
I will make your requested changes once I get back home.

**Comment by daPhipz** [comment]
Done!

**Comment by daniellandau** [comment]
Great, thanks!

**Pull request merged**